.github/workflows/ccpp.yml

```diff
@@ -1,28 +1,311 @@
+      run: CFLAGS="-fPIC" CXXFLAGS="-fPIC" ./configure
+    - name: make
+      run: make
+    - name: swig bindings
+      run: cd pjsip-apps/src/swig && make
+
+  build-ubuntu-default-full-bundle:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: install dependencies
+      run: sudo apt-get install -y swig sip-tester libopencore-amrnb-dev
+    - name: config site
+      run: cd pjlib/include/pj && cp config_site_test_full.h config_site.h
+    - name: configure
+      run: CFLAGS="-fPIC" CXXFLAGS="-fPIC" ./configure
+    - name: make
+      run: make
+    - name: swig bindings
+      run: cd pjsip-apps/src/swig && make
+    - name: unit tests
+      run: make pjlib-test pjlib-util-test pjmedia-test pjsua-test
+
+  build-ubuntu-default-full-bundle-2:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: install dependencies
+      run: sudo apt-get install -y libopencore-amrnb-dev
+    - name: config site
+      run: cd pjlib/include/pj && cp config_site_test_full.h config_site.h
+    - name: configure
+      run: CFLAGS="-fPIC" CXXFLAGS="-fPIC" ./configure
+    - name: make
+      run: make
+    - name: unit tests
+      run: make pjnath-test
+
+  build-ubuntu-default-full-bundle-3:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: install dependencies
+      run: sudo apt-get install -y libopencore-amrnb-dev
+    - name: config site
+      run: cd pjlib/include/pj && cp config_site_test_full.h config_site.h
+    - name: configure
+      run: CFLAGS="-fPIC" CXXFLAGS="-fPIC" ./configure
+    - name: make
+      run: make
+    - name: unit tests
+      run: make pjsip-test
+
+  build-ubuntu-no-tls:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: install dependencies
+      run: sudo apt-get install -y swig
+    - name: configure
+      run: CFLAGS="-fPIC" CXXFLAGS="-fPIC" ./configure --disable-ssl
+    - name: make
+      run: make
+    - name: swig bindings
+      run: cd pjsip-apps/src/swig && make
+
+  build-ubuntu-gnu-tls:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: install dependencies
+      run: sudo apt-get update && sudo apt-get install -y --fix-missing swig libgnutls28-dev
+    - name: configure
+      run: CFLAGS="-fPIC" CXXFLAGS="-fPIC" ./configure --with-gnutls=/usr/
+    - name: make
+      run: make
+    - name: swig bindings
+      run: cd pjsip-apps/src/swig && make
+
+  build-ubuntu-video-openh264-1:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: install dependencies
+      run: sudo apt-get install -y swig nasm sip-tester libvpx-dev libopencore-amrnb-dev
+    - name: get openh264
+      run: git clone --single-branch --branch openh264v2.1.0 https://github.com/cisco/openh264.git
+    - name: build openh264
+      run: cd openh264 && make && sudo make install && sudo rm /usr/local/lib/libopenh264*.so*
```

Comment:
why libopenh264.so is removed here? better add note/comment about it?
if only static lib is wanted, perhaps there is configure option for that, e.g: usually kind of `--disabled-shared`?